src/solutions/components/genai/Chat.js

```diff
@@ -0,0 +1,155 @@
+import React, { useState } from 'react';
+
+import ForumIcon from '@mui/icons-material/Forum';
+import CloseIcon from '@mui/icons-material/Close';
+import OpenInFullIcon from '@mui/icons-material/OpenInFull';
+import CloseFullscreenIcon from '@mui/icons-material/CloseFullscreen';
+import { Button } from '@mui/material';
+
+import { getDynamicConfigValue } from '../../config/dynamicConfig';
+
+export default function Chat(props) {
+  const urlParams = new URLSearchParams(window.location.search);
+  const chatEnabledUrlString = urlParams.get('genAiChatEnabled');
+  const genAiUrlString = urlParams.get('genAiUrl');
```

Comment:
Would be cool to do a 'true' embedded GenAI chat interface, rather than an external iFrame.
We have some ideas here from MH, lets disscuss!

Reply:
taking this out for now